yt/data_objects/level_sets/clump_handling.py

```diff
@@ -67,6 +64,7 @@ def __init__(self, data, field, parent=None,
         self.min_val = self.data[field].min()
         self.max_val = self.data[field].max()
         self.info = {}
+        self.children = []
```

Comment:
Yes, it's much clearer to add an empty one in the initializer instead of assuming it'll get created later